modules/ppcp-compat/src/PPEC/PpecHelper.php

```diff
@@ -11,7 +11,7 @@
  * Helper class with various constants associated to the PayPal Express Checkout plugin, as well as methods for figuring
  * out the status of the gateway.
  */
-class PPECHelper {
+class PpecHelper {
```

Comment:
Would something break if we keep this class as `PPECHelper`. Keep in mind that `PPEC` is an acronym (PayPal Express Checkout) and it looks a bit weird capitalized that way.
Alternatively, we could just rename it to `PayPalCheckoutHelper` or something like that.

Reply:
✔️ 40b5783